Data-tranformations/fairgenomes/analyses.yarrrml.yaml

```diff
@@ -0,0 +1,26 @@
+prefixes:
+  ega.dataset: https://www.ebi.ac.uk/ega/datasets/
```

Comment:
This is not used and could be replaced with:
I found some URL patterns that can be used to get metadata records from EGA that we could use as the `subject` in our mappings (see the Querying by Object heading at [Metadata Rest Endpoints](https://ega-archive.org/discovery/metadata/public-metadata-api/)):
```
  ega.analyses: https://metadata.ega-archive.org/analyses/
```